app/workers/education_form/create_daily_spool_files.rb

```diff
@@ -147,5 +163,13 @@ def track_form_type(type, rpo)
     def stats
       @stats ||= Hash.new(Hash.new(0))
     end
+
+    def log_info(message)
+      if Flipper.enabled?(:sidekiq_create_daily_spool_file_logging)
+        log_exception_to_sentry(DailySpoolFileLogging.new(message))
```

Comment:
The parameters for `log_exception_to_sentry` are `(exception, extra_context = {}, tags_context = {}, level = 'error')`. and it'll log the message to the rails log.

Reply:
Removed feature flag and set log level to `:info` for this method